src/components/subsetDraft/Classification.jsx

```diff
@@ -252,11 +261,11 @@ export const CodeInfo = ({id, item, onChange}) => {
                 {!item.notes
                     ? <Text>{t('Notes are not found.')}</Text>
                     : item.notes.map(note => (
-                        <div style={{
+                        <div key={note} style={{
                             padding: '10px 50px 20px 50px'
                         }}>
                             <Title size={4}>{t('Notes')}</Title>
-                            <Paragraph style={{width: '65%'}}>{note.note}</Paragraph>
+                            <div style={{width: '65%'}} className="ssb-paragraph" dangerouslySetInnerHTML={ {__html: replaceRefWithBold(note.note)} } />
```

Comment:
Is it safe?

Reply:
No, you might be right about this. We should sanitize this before inserting it.